qutebrowser/browser/webengine/langs.tsv

```diff
@@ -0,0 +1,42 @@
+af-ZA	Afrikaans (South Africa)	af-ZA-3-0.bdic
+bg-BG	Bulgarian (Bulgaria)	bg-BG-3-0.bdic
+ca-ES	Catalan (Spain)	ca-ES-3-0.bdic
+cs-CZ	Czech (Czech Republic)	cs-CZ-3-0.bdic
+da-DK	Danish (Denmark)	da-DK-3-0.bdic
+de-DE	German (Germany)	de-DE-3-0.bdic
+el-GR	Greek (Greece)	el-GR-3-0.bdic
+en-CA	English (Canada)	en-CA-7-1.bdic
+en-GB	English (United Kingdom)	en-GB-7-1.bdic
+en-US	English (United States)	en-US-7-1.bdic
+es-ES	Spanish (Spain)	es-ES-3-0.bdic
+et-EE	Estonian (Estonia)	et-EE-3-0.bdic
+fa-IR	Farsi (Iran)	fa-IR-7-0.bdic
+fo-FO	Faroese (Faroe Islands)	fo-FO-3-0.bdic
+fr-FR	French (France)	fr-FR-3-0.bdic
+he-IL	Hebrew (Israel)	he-IL-3-0.bdic
+hi-IN	Hindi (India)	hi-IN-3-0.bdic
+hr-HR	Croatian (Croatia)	hr-HR-3-0.bdic
+hu-HU	Hungarian (Hungary)	hu-HU-3-0.bdic
+id-ID	Indonesian (Indonesia)	id-ID-3-0.bdic
+it-IT	Italian (Italy)	it-IT-3-0.bdic
+ko	Korean	ko-3-0.bdic
+lt-LT	Lithuanian (Lithuania)	lt-LT-3-0.bdic
+lv-LV	Latvian (Latvia)	lv-LV-3-0.bdic
+nb-NO	Norwegian (Norway)	nb-NO-3-0.bdic
+nl-NL	Dutch (Netherlands)	nl-NL-3-0.bdic
+pl-PL	Polish (Poland)	pl-PL-3-0.bdic
+pt-BR	Portuguese (Brazil)	pt-BR-3-0.bdic
+pt-PT	Portuguese (Portugal)	pt-PT-3-0.bdic
+ro-RO	Romanian (Romania)	ro-RO-3-0.bdic
+ru-RU	Russian (Russia)	ru-RU-3-0.bdic
+sh	Serbo-Croatian	sh-3-0.bdic
+sk-SK	Slovak (Slovakia)	sk-SK-3-0.bdic
+sl-SI	Slovenian (Slovenia)	sl-SI-3-0.bdic
+sq	Albanian	sq-3-0.bdic
+sr	Serbian	sr-3-0.bdic
+sv-SE	Swedish (Sweden)	sv-SE-3-0.bdic
+ta-IN	Tamil (India)	ta-IN-3-0.bdic
+tg-TG	Tajik (Tajikistan)	tg-TG-5-0.bdic
+tr-TR	Turkish (Turkey)	tr-TR-4-0.bdic
+uk-UA	Ukrainian (Ukraine)	uk-UA-3-0.bdic
+vi-VN	Vietnamese (Viet Nam)	vi-VN-3-0.bdic
```

Comment:
I don't really like the duplication here to be honest - I think the code inside qutebrowser could use the information from `configdata` instead, and the download script could use the Google API, I'll comment on the script below.